app/lib/brexit_checker/validators/group_validator.rb

```diff
@@ -0,0 +1,24 @@
+class BrexitChecker::Validators::GroupValidator < ActiveModel::Validator
```

Comment:
This is a _much_ better idea, not sure why we didn't do this last year already! Nice one!